addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -124,6 +149,7 @@ var WysiwygMultizone = Wysiwyg.extend({
         this.savingMutex = new concurrency.Mutex();
         this._super(parent, options);
         this._onChangeThrottled = _.throttle(this._onChange.bind(this), 300);
+        this.snippetsEditor = new SnippetsEditor.Class(this, options);
```

Comment:
Why are you adding this?

Reply:
I can't access to `toggleMegaMenuSnippets` without this :open_mouth:

Reply:
nevermind...

Reply:
nevermind...